lang/dart.md

```diff
@@ -0,0 +1,189 @@
+```
+
+**Step - 2:**
+
+ Type the following command in terminal to install the Dart SDK using apt-get option.
+
+```bash
+sudo apt-get update  
+sudo apt-get install dart
+```
+
+It will successfully download the Dart SDK.
+
+**Installation a Debian Package:**
+
+We can download Dart SDK as a Debian package in the .deb package format. To make all Dart binaries accessible, we have to change the PATH by typing the following command.
+
+```bash
+export PATH="$PATH:/usr/lib/dart/bin"
+```
+
+To change the PATH for upcoming terminal sessions, use a below command:
+
+```bash
+echo 'export PATH="$PATH:/usr/lib/dart/bin"' >> ~/.profile
+```
+
+## Popular Dart IDEs/editors 
+- The Eclipse,VsCode, IntelliJ, and WebStorm are the IDEs from the Jet brains that support the Dart Programming, but WebStorm is more popular than others. We can download it from <https://www.jetbrains.com/webstorm/download/>.
+
+## Some Cool projects in Dart
+
+**1. [Invoice Naija](https://github.com/invoiceninja/admin-portal) :** This is a business application for documenting invoices, tasks, and expenses.
+It was built using flutter and laravel.
+- Invoice Naija
+- Extra Features
+- Clients
+- Products
+- Quotes
+- Payments
+ 
+
+**2. [Bit Alarm](https://github.com/damoonrashidi/bitalarm) :** This is a cryptocurrency application created to keep track of different cryptocurrencies
+- Bit Alarm
+- Extra Features
+- Loading indicator.
+- Graph out historical data for a currency
+- Scan QR-code to add a wallet to wallet list
+- Remove wallet from the list
+- Dynamic portfolio based on the address (ETH + ERC20-tokens, LTC, BTC, BCH, DASH and ADA for now)
+- Further, Ability to add individual assets in addition to wallets
+
+**3 [Weight Tracker](https://github.com/MSzalek-Mobile/weight_tracker) :** Weight Tracker is an application dedicated to people who want to dump or maintain weight. It allows you to keep an eye on your progress and pursue your weight goals.
+- **Screenshot Image**
+- **Extra Features**
+- **Simple**
+- **User-Friendly** 
+
+**4. [Flutter Music Player](https://github.com/iampawan/Flutter-Music-Player) :**
+Flutter Music Player is an open-source music player application, available only on Andriod, for now.
+- Screenshot Image
+- Extra Features
+- Fetch local songs (Inbuilt Permission)
+- Beautiful UI with multiple themes(Coming Soon)
+- Animation
+- Full Fledged Example
+- play (local files)
+- stop
+- pause
+- seek
+- shuffle
+- Album Art
+- onComplete
+- onDuration / onCurrentPosition
+
+**5. [Trovami](https://github.com/Samaritan1011001/Trovami) :**
+Trovami is a flutter based android application, you can use to share your current location with other persons.
+- preview
+- Extra Features
+- Create groups and choose to share your live location
+- View group members on the map
+- Google Sign-in
+ 
+**Thanks For Reading**
```

Comment:
```suggestion
**Thanks For Reading!**
```

Reply:
It really should be 👇 

```
**Thanks for reading!**
```

I cannot see who you've used title case here 😅